### DISCUSSION TITLE
虽然这个项目带了一些活，但是它任然是一个正经的项目

### DISCUSSION THREAD

**Discussion by baiwusanyu-c** [ikun-svelte/ikun-ui#10] [Announcements]
虽然这个项目带了一些整活性质，但是它任然是一个正经的项目。
先叠个甲！！！这个项目它没有对任何人的攻击和恶意
同时如果你有好的 idea 欢迎讨论与PR

**Comment by Maicarons** [comment]
需要vue版，急急急，我是只因国王

**Reply by so1ve** [reply]
社区可以自己移植吧，白雾做这个项目应该是觉得svelte没什么现成组件库

**Comment by testEcho9527** [comment]
看了这个项目后，我的铁山靠又进阶了一层了

**Comment by greenhand-xj** [comment]
确实需要一些svelte的UI库

**Comment by blazer233** [comment]
太6了，火钳刘明

**Comment by wanniwa** [comment]
居然没有提供kunkun 专属图标，不是很符合表项目名称

**Comment by Wxw-Gu** [comment]
不是ikun 可以用库么

**Reply by so1ve** [reply]
chick-ui

**Comment by haoziqaq** [comment]
KunKun 应该拥有自己的 Lisence 而不是 MIT !

**Reply by Maicarons** [reply]
绿尸寒是吧😅

**Comment by greenhand-xj** [comment]
这是想要快速的得到star吗，这项目名后面肯定要改的😂

**Comment by chen-haojie0525** [comment]
那我觉得，这件事情，泰酷辣！！！

**Comment by so1ve** [comment]
苏更，立刻学习svelte

**Reply by baiwusanyu-c** [reply]
ray 佬 ！您來搞一個！🥵

**Reply by so1ve** [reply]
@baiwusanyu-c Make ikun-ui great again 😠

**Comment by ThrowSyntaxError** [comment]
巅峰产生虚伪的拥护，黄昏见证真正的信徒，坤门！

**Reply by Maicarons** [reply]
巅峰产生虚伪的拥护，黄昏见证真正的信徒，坤门！

**Comment by SCPCBGame** [comment]
肯定是纯国产![Cache_5f4b8f54e0a6a8b7.jpg](https://github.com/ikun-svelte/ikun-ui/assets/139687132/ded279b5-5f17-43cc-842b-e7e52980b596)
